tests/testthat/test-covariance_structures.r

```diff
@@ -1,7 +1,7 @@
 
 test_that("unit test for unstructured covariances", {
   
-  for(i in seq_len(100L)){
+  for(i in seq_len(20L)){
```

Comment:
On second thought, it will be better to do `for (nc in c(0:4, 16L, 64L, 256L))`, so that you are sure to test the important edge cases and also some big ones.